notebooks/work-in-progress/VCERARE.ipynb

```diff
@@ -0,0 +1,288 @@
+    "\n",
+    "See the [Zenodo archive README](https://zenodo.org/records/13937523) for more detail on the raw data schema and contents. \n",
+    "\n",
+    "### Processed PUDL Parquet files (S3 bucket)\n",
+    "\n",
+    "The processed data are published as Apache Parquet files--a file format designed for efficient data storage and retreival. These processed files combine all years and generation sources into one table. They also restructure the data from a wide format to a tall format by pulling county into a column value rather than a column header. Additional informational columns are added to the processed data including: \n",
+    "\n",
+    "* Latitude\n",
+    "* Longitude\n",
+    "* County FIPS code\n",
+    "* Report rear\n",
+    "* Datetime UTC\n",
+    "* Separate state and county (or Lake) name fields (the raw data is formatted together as county_state)\n",
+    "\n",
+    "Take a look at our [data dictionary](https://catalystcoop-pudl.readthedocs.io/en/nightly/data_dictionaries/pudl_db.html#out-vcerare-hourly-available-capacity-factor) for more information on the processed table schema."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0e861aa-01c8-4174-bac3-4e418b880f9a",
+   "metadata": {},
+   "source": [
+    "## Which one should you use?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "473c509e-f518-4159-85f3-eb6764e980ca",
+   "metadata": {},
+   "source": [
+    "If you want or need to alter the layout of the data in order to feed it into your model, are interested in a particular subset of the data, want to connect it to geospatial data, or want to view / run analysis on all the data at once, the processed PUDL Parquet files are the way to go! For context, the row limit in Excel is a little more than 1 million rows. The processed PUDL table contains 136,437,000 rows. Below, we'll show you how to wrangle this dataset without blowing up your computer."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c399ea58-aa9b-45e1-9f0b-57adbc8d949e",
+   "metadata": {},
+   "source": [
+    "## Accessing RARE Parquet files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "77ef0416-3f43-48f2-b71a-76a383006f2c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import io\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import random\n",
+    "import sys\n",
+    "import pudl\n",
+    "pudl_paths = pudl.workspace.setup.PudlPaths()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "49394cc9-116b-4fee-a445-02afcc3215b1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rare_df = pd.read_parquet(\n",
+    "    pudl_paths.pudl_output / \"parquet/out_vceregen__hourly_available_capacity_factor.parquet\"\n",
+    ")\n",
```

Comment:
For me this crashed the kernel several times. Adding `engine=pyarrow` and dtype_backend='pyarrow' here might be helpful, with a short explanation of why.

E.g., 

```
rare_df = pd.read_parquet(
    pudl_paths.pudl_output / "parquet/out_vcerare__hourly_available_capacity_factor.parquet", engine='pyarrow', dtype_backend='pyarrow'
)
```